components/raftstore/src/store/config.rs

```diff
@@ -92,6 +92,9 @@ pub struct Config {
     #[online_config(skip)]
     pub raft_reject_transfer_leader_duration: ReadableDuration,
 
+    // the percentage used capacity of raft engine to reach `almost full` disk stat
+    pub raft_engine_almost_full_percent: u64,
```

Comment:
As you mentioned today we use reserve-space for kv disk protection. I think it would be consistant to mirror the name: reserve-raft-disk-space for raft disk.  And same as kv disk if it's zero, then raft disk protection is disabled.

Reply:
This may introduce some works, I will take a look after my vacation.